burnman/minerals/other.py

```diff
@@ -12,10 +12,34 @@
 
 from .. import mineral_helpers as helpers
 from ..mineral import Mineral
+from ..solidsolution import SolidSolution
+from ..solutionmodel import *
 
 from .SLB_2011 import periclase, wuestite, mg_perovskite, fe_perovskite
 
-
+class make_endmember(SolidSolution):
```

Comment:
The alternative is to write a new EoS that takes a set of minerals and adds together their properties. Perhaps it would be cleanest to write a dummy EoS that accesses all the functions in mechanical solution, allowing the class to be written as class MakeEndmember(Mineral):

A final possibility is to write a function make_endmember that returns an instance of class MadeEndmember(Mineral). I honestly don't know which alternative is best.